main.go

```diff
@@ -12,25 +12,27 @@ import (
 	"time"
 
 	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
 	"github.com/prometheus/common/log"
 )
 
-const applicationName = "opentsdb_exporter"
+const (
+	applicationName = "opentsdb_exporter"
+	metricsRoute    = "/metrics"
+	probeRoute      = "/opentsdb"
```

Comment:
I wondered that, but it's not really a probe either. The SNMP exporter uses `/snmp` or similar, so at least this is consistent.